README.md

```diff
@@ -722,6 +722,28 @@ Req/Bytes counts sampled once per second.
 125k requests in 10.03s, 43.7 MB read
 ```
 
+## Corner cases resolutions
+
+It could be possible, especially if we are working with Federation, to come across different return types, for instance, String instead of Int, etc. [Here](https://github.com/mercurius-js/cache/issues/128) is the relative issue.
```

Comment:
It could be possible, especially when working with Federation, to encounter varying return types. For instance, one might come across a situation where a String is returned instead of an Int. [Here](https://github.com/mercurius-js/cache/issues/128) is the related issue.